moviesurfer/cli/main.py

```diff
@@ -1,2 +1,28 @@
-def main():
-    print("Hello from CLI, functionality is to be added")
+import logging
+
+import click
+
+from moviesurfer.data.download_data import download_data
+
+logger = logging.getLogger(__name__)
+
+
+@click.group(chain=True)
```

Comment:
why `chain=True`?
Do you suggest multiple sub-commands to be run at once?
e.g. `moviesurfer data download preproccess split`
imho, it can lead to to unnecessary complications